duckduckgo/iOS
# Ignore email JS pixel events

Task/Issue URL: https://app.asana.com/0/414235014887631/1203393655574807/f
Tech Design URL:
CC:

**Description**:

This PR updates the iOS app to account for email JS pixel events. The iOS app sends its own email autofill pixels, so those coming from the JS layer should be ignored.



**Steps to test this PR**:
1. Test the email autofill feature and verify that you don't see double pixels being fired



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
